functions/src/mappers/github/repository.mapper.ts

```diff
@@ -24,6 +30,9 @@ export interface GitHubRepositoryModel {
   pullRequests?: GitHubPullRequestModel[];
   events?: GitHubEventModel[];
   releases?: GitHubReleaseModel[];
+  issues?: GitHubIssueModel[];
+  contributors: GitHubContributorModel[];
```

Comment:
I think `contributors` needs to be optional

Reply:
Fixed